### PR TITLE
Set the developer key header to both the underscored and hyphenated v…

### PR DESCRIPTION
**Pull request by coryverta**
…ersion

This works around nginx dropping underscored header names.

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java [hunk 1]
```diff
@@ -68,10 +68,13 @@ private Metadata getMetadataHeaders() {
       Metadata.Key<String> email_key = Metadata.Key.of("email", Metadata.ASCII_STRING_MARSHALLER);
       Metadata.Key<String> dev_key =
           Metadata.Key.of("developer_key", Metadata.ASCII_STRING_MARSHALLER);
+      Metadata.Key<String> dev_key_hyphen =
+          Metadata.Key.of("developer-key", Metadata.ASCII_STRING_MARSHALLER);
       Metadata.Key<String> source_key = Metadata.Key.of("source", Metadata.ASCII_STRING_MARSHALLER);
 
       requestHeaders.put(email_key, this.serviceUserEmail);
       requestHeaders.put(dev_key, this.serviceUserDevKey);
+      requestHeaders.put(dev_key_hyphen, this.serviceUserDevKey);
```
Comment (coryverta) [inline backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java:77]: This inserts the hyphenated key in addition to the underscored key

**Review by convoliution — APPROVED** [review]

**Review by conradoverta — APPROVED** [review]
Let's make sure we have a similar change on top of the required tag.

**Comment by coryverta** [comment]
I set the upstream branch to the last PR branch that was merged into master, is there a better way to do this?  Master does not appear to have a tag for the merge.

**Comment by conradoverta** [comment]
> I set the upstream branch to the last PR branch that was merged into master, is there a better way to do this? Master does not appear to have a tag for the merge.

Oh, cool. That's fine. It just wasn't clear =) We can create a base branch from that commit so that we can keep track of just the specific changes to facilitate review.

**Pull request merged**